cmd/vicadmin/vicadm.go

```
@@ -361,6 +390,11 @@ func findDatastoreLogs(c *session.Session) (map[string]entryReader, error) {
 		return nil, errors.New(detail)
 	}
 
+	err = addApplianceLogs(ctx, c, readers)
+	if err != nil {
+		log.Errorf("Issue collecting appliance logs: %s", err)
+	}
```

Guessing we are swallowing this error by purpose